x-pack/plugins/watcher/public/models/execute_details/execute_details.js

```diff
@@ -47,9 +48,16 @@ export class ExecuteDetails {
 
   formatTime(timeUnit, timeValue) {
```

Comment:
Minor nit: can we extract this logic into a standalone function that's not part of the class? By exposing it as a public member, it looks like we intend for it to be consumed by consumers of this class, but it doesn't appear that's the case.

Reply:
yep, fixed.